numpy/lib/tests/test_arraysetops.py

```diff
@@ -1226,8 +1231,14 @@ def test_unique_array_api_functions(self):
             )
         ]:
             assert len(res_unique_array_api) == len(res_unique)
+            if not isinstance(res_unique_array_api, tuple):
+                res_unique_array_api = (res_unique_array_api,)
+            if not isinstance(res_unique, tuple):
+                res_unique = (res_unique,)
+
             for actual, expected in zip(res_unique_array_api, res_unique):
-                assert_array_equal(actual, expected)
+                # Order of output is not guaranteed
+                assert_equal(np.sort(actual), np.sort(expected))
 
```

Comment:
Can we make sure we have tests for both NaNs and `-0.0` vs `0.0`.  For NaN, test with `np.nan` and `-np.nan` (I hope `-np.nan` is enough, but it stays in python world).